fe/fe-core/src/main/java/com/starrocks/catalog/CatalogName.java

```diff
@@ -0,0 +1,50 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.google.common.base.Preconditions;
+
+import java.util.Objects;
+
+public final class CatalogName
```

Comment:
why not use String as Catalog name directly?

Reply:
The main reason is that using string as a key is a bit bare, and using Catalogname is more elegant.I wonder if it's more convenient to use String. should I change back to String？

Reply:
Both ways are OK for me

Reply:
done, I'd better use String first. It looks more convenient